src/main/java/io/strimzi/kafka/bridge/amqp/AmqpSinkBridgeEndpoint.java

```diff
@@ -50,7 +53,7 @@
 	private static final String GROUP_ID_MATCH = "/group.id/";
 	
 	// converter from ConsumerRecord to AMQP message
-	private MessageConverter<K, V, Message> converter;
+	private MessageConverter<K, V, Message, Collection<Buffer>> converter;
```

Comment:
It needs to be a `Collection<Message>` (you are using `Buffer` for HTTP not AMQP)